<div>
<br>
<hr>
<table width=100% align="center" cellpadding="10" cell spacing="0">
     <tr>
     <td>
<h3>References</h3>
<ol>
     <li> Martin JS, Xu Z, Reiner AP, Mohlke KL, Sullivan P, Ren B, Hu M, and Li Y. (2017) HUGIn: Hi-C Unifying Genomic Interrogator. bioRxiv 117531; doi: <a href="https://doi.org/10.1101/117531" target="_blank">https://doi.org/10.1101/117531</a> .</li>
<li> Schmitt AD, Hu M, Jung I, Xu Z, Qiu Y, Tan CL, Li Y, Lin S, Lin Y, Barr CL and Ren B. (2016) A compendium of chromatin contact maps reveal spatially active regions in the human genome. Cell Reports. 17(8):2012-2059.</li>
     <li>Xu Z, Zhang G, Wu C, Li Y and Hu M. (2016) FastHiC: a fast algorithm to detect long-range chromosomal interactions from Hi-C data. Bioinformatics. 32(17): 2692-2695. </li>
     <li>Xu Z, Zhang G, Jin F, Chen M, Furey TS, Sullivan PF, Qin ZS, Hu M and Li Y. (2016) A hidden Markov random field based Bayesian method for the detection of long-range chromosomal interactions in Hi-C data. Bioinformatics. 32(5): 650-656. </li>
<li>Jin F, Li Y, Dixon JR, Selvaraj S, Ye Z, Lee AY, Yen CA, Schmitt AD, Espinoza CA and Ren B (2013) A high-resolution map of the three-dimensional chromatin interactome in human cells. Nature 503: 290-294.</li>
<li>Ay F, Bailey TL and Noble WS (2014) Statistical confidence estimation for Hi-C data reveals regulatory
chromatin contacts. Genome Research 24: 974-988.</li>
</ol>
</td>
<td>
<a href="http://info.flagcounter.com/royF" target="_blank"><img src="http://s01.flagcounter.com/count2/royF/bg_FFFFFF/txt_000000/border_000000/columns_2/maxflags_10/viewers_0/labels_0/pageviews_1/flags_0/percent_0/" alt="Flag Counter" border="0"></a>
</td>
</tr>
</table>
<p>Questions or comments about HUGIN, please contact jsmartin at email.unc.edu.</p>

</div>
                                            
</body>
</html>